internal/hcs/process.go

```diff
@@ -163,7 +163,30 @@ func (process *Process) Kill(ctx context.Context) (bool, error) {
 		return true, nil
 	}
 
-	resultJSON, err := vmcompute.HcsTerminateProcess(ctx, process.handle)
+	// HCS serializes the signals sent to the target pid on a per client basis.
```

Comment:
What exactly does "per client" mean here? Does it serialize per compute system handle?

Reply:
Yes, that is was I meant. I can change to "per compute system handle"